### PR TITLE
Ferret visual areas addition

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/uberon/issues/2352
Tagging @stevevanhooser & @daniscafidi
Will remove CSV after this is fully ready 
Things to fix - Noticed that there isn't and dc:contributor 
@stevevanhooser & @daniscafidi - could you pass me your ORCIDS please? Will make sure you get attribution for your contribution :)

### CODE REVIEW

**Comment by shawntanzk** [comment]
I'll also fix the defs to be more ontology conformant

**Comment by shawntanzk** [comment]
The following 2 terms have the same label (and look the same in general except for the synonym), could you clarify @daniscafidi  (its what is causing the qc failure atm) 

ID | Label | Superclass | Part of axioms | Taxon | Subclass axioms | Exact Synonym | Exact Synonym Reference | Definition | Definition Reference | Comment
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
UBERON:8440064 | spenial visual area (sensu Mustela putorius furo) | UBERON:0035014 | UBERON:0002104 | NCBITaxon:9669 | capable of part of' some 'visual perception' | SVA (sensu Mustela putorius furo) | PMID:20853515 | A visual area in the ferret located on the dorsal bank of the splenial sulcus between the medial border of area 17 and the lateral borter of CVA. | PMID:20853515\|DOI:10.1007/978-3-319-76626-3
UBERON:8440065 | spenial visual area (sensu Mustela putorius furo) | UBERON:0035014 | UBERON:0002104 | NCBITaxon:9669 | capable of part of' some 'visual perception' | splenial sulcal cortex (sensu Mustela putorius furo)\|SPLC (sensu Mustela putorius furo) | PMID:20853515 | A visual area in the ferret located on the dorsal bank of the splenial sulcus between the medial border of area 17 and the lateral borter of CVA. | PMID:20853515\|DOI:10.1007/978-3-319-76626-3

**Comment by daniscafidi** [comment]
Hi Shawn, I imagine it’s an error on my part. I pulled out some of the terms while I was ensuring each synonym did not have a unique citation as you’d indicated that the reference must apply to all in that row. I must have not collapsed them back together after.

**Comment by shawntanzk** [comment]
@daniscafidi so does that mean I should merge those two terms together (all the synonyms apply to the term)? Thanks

**Comment by daniscafidi** [comment]
Yes!

**Comment by shawntanzk** [comment]
UBERON:8440063
Screenshot 2022-06-18 at 16 09 08
UBERON:8440050
Screenshot 2022-06-18 at 16 10 01

**Comment by stevevanhooser** [comment]
Thanks @shawntanzk. I don't quite understand the errors but let us know if you need any input in navigating it.

Best
Steve

**Comment by shawntanzk** [comment]
@stevevanhooser dont worry about, will ask if I need any input from you all. What would be useful is to have both your orcids. Thanks!

**Comment by shawntanzk** [comment]
@dosumis need some help with this:
for UBERON:8440063 case 
I think the owl:Nothing disjointness between visual system and vestibulo-auditory system might be tricky with this case?
for UBERON:8440050 case
I think somatic sensory system being part of somatic nervous system might be wrong (based on defs) as somatic sensory system should surely include the CNS parts that are involved in somatic sensory. somatic nervous system clearly only defines it as peripheral part of it though

**Comment by stevevanhooser** [comment]
Hi - my ORCID is 0000-0002-1112-5832

@daniscafidi - you can create one on https://orcid.org/ . It just takes a few seconds. 

Thanks
Steve

**Comment by shawntanzk** [comment]
Alternative plan: 
will change the offending axioms to capable_of_part_of some X instead or part_of some X system. 

Upside, it should fix everything, keeping the logical axioms consistent with defs.
Downside, I'm not sure if people would expect that part of the brain to show up when querying the systems

**Comment by shawntanzk** [comment]
I've changed the part_of X system to capable_of_part_of which also fits the def better.
With this I don't think there is a NEED to change any of the disjointness of logical def of others, but still have questions on if we SHOULD change them

**Comment by stevevanhooser** [comment]
Q: if someone wants to do a search for all areas that are part of some visual cortex, will it still work?

**Comment by shawntanzk** [comment]
> Q: if someone wants to do a search for all areas that are part of some visual cortex, will it still work?

The terms that have '`part of' some 'visual cortex`' will come up, but not the ones which only have `'capable of part of' some 'visual perception'`. 
Visual cortex does not have an equivalent class axiom, so everything that you want to be under there has to be asserted.

**Review by dosumis — CHANGES_REQUESTED** [review]
1. I'm still seeing some duplicates in the ROBOT template, e.g. 

ID | Label | Superclass | Part of axioms | Taxon | Subclass axioms | Exact Synonym | Exact Synonym Reference | Definition | Definition Reference | Comment
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  | ID | LABEL | SC % | SC part_of some % SPLIT=\| | SC 'in taxon' some % | SC % SPLIT=\| | A oboInOwl:hasExactSynonym SPLIT=\| | >A oboInOwl:hasDbXref SPLIT=\| | A IAO:0000115 | >A oboInOwl:hasDbXref SPLIT=\| | A rdfs:comment
  | UBERON:8440050 | posterior parietal rostral cortical area (sensu Mustela putorius furo) | UBERON:0035014 | UBERON:0002104\|UBERON:0003942\|UBERON:0000411 | NCBITaxon:9669 | 'capable of part of' some 'visual perception' | PPr (sensu Mustela putorius furo) | PMID:12427679 | A multimodal area of the ferret involved in vision and somatosensation that lies between the medial wall of the hemisphere and the medial lip of the suprasylvian sulcus and is posterior to caudal somatosensory cortex (SIII) and anterior to PPc. | PMID:12427679 |  
  | UBERON:8440051 | posterior parietal rostral cortical area (sensu Mustela putorius furo) | UBERON:0035014 | UBERON:0002104\|UBERON:0003942\|UBERON:0000411 | NCBITaxon:9669 | 'capable of part of' some 'visual perception' | posterior parietal cortex, rostral part (sensu Mustela putorius furo) | DOI:10.1007/978-3-319-76626-3 | A multimodal area of the ferret involved in vision and somatosensation that lies between the medial wall of the hemisphere and the medial lip of the suprasylvian sulcus and is posterior to caudal somatosensory cortex (SIII) and anterior to PPc. | PMID:12427679 |  

2. Where does this come from?  It looks wrong:

image

3. Assume what you mean here "Remove owl:Nothing disjointness between visual system and vestibulo-auditory system"  is to remove this axiom:  
image
In plain English - nothing is part of the visual system and the vestibulo-auditory system.  
 I suspect that, if justified at all, this based on a morphological subdivision of cortex that no longer applies when we define regions functionally.

**Comment by shawntanzk** [comment]
> I'm still seeing some duplicates in the ROBOT template

Oh ignore the template, I made the changes in the edit file itself. I will be removing the template file as this is a one off addition rather than an ongoing template file. 

> Where does this come from? It looks wrong:

That is an asserted axiom - I think it is right as that region is also has somatosensory function - however, as per https://github.com/obophenotype/uberon/pull/2534#issuecomment-1161288180 I have changed these to capable_of_part_of some GO BP -> it is why there is no more owl:Nothing error. However this means that these regions wont be inferred as part_of the somatic sensory system which is a larger problem of spatial vs functional definition I guess, which is your point 3. 

I think, for now, this can all pass (assuming you are alright with all the content in the PR) - but the problem with spatial vs functional -> which does come back to this ticket https://github.com/obophenotype/uberon/issues/2411

**Comment by dosumis** [comment]
> Oh ignore the template, I made the changes in the edit file itself. I will be removing the template file as this is a one off addition rather than an ongoing template file.

- Ahh - was using that as a convenient way to review.  I guess I should use the diffs.  Having looked at those and the rest of the issues here - maybe best if we find some time for you to talk me through changes.  I think there are more opportunities for patternization that we should really take advantage of.

**Comment by shawntanzk** [comment]
@dosumis made all the fixes we talked about - would like a proper review before merging, thanks!

**Comment by shawntanzk** [comment]
Will merge for now and open a ticket/or go through the linked ticket and add questions to it.

**Comment by dosumis** [comment]
Yeh - I think we should drop spatial disjointness of sensory systems.  I don't think it makes any sense (BTW - using go sensory perception terms would align with CL and FBbt and allow us to auto-populate general functional systems).
